pymongo/hello.py

```diff
@@ -218,3 +218,7 @@ def service_id(self) -> Optional[ObjectId]:
     @property
     def hello_ok(self) -> bool:
         return self._doc.get("helloOk", False)
+
+    @property
+    def connection_id(self) -> Optional[ObjectId]:
```

Comment:
We already have a [public `connection_id` concept](https://pymongo.readthedocs.io/en/stable/api/pymongo/monitoring.html#pymongo.monitoring.ConnectionCheckedOutEvent.connection_id) so unfortunately I think we need to rename this. Perhaps `server_connection_id`?

Reply:
Oh I didn't realize this was on Hello. Ignore this comment, connection_id is fine here.